### DISCUSSION TITLE
Update teampass.net Website

### DISCUSSION THREAD

**Discussion by bjinthahouse** [nilsteampassnet/TeamPass#3916] [Ideas]
Hi @nilsteampassnet,

just a small pointer from my side: You should definitely update the Teampass website. I work at an IT services company and the decision to use (or keep using) a tool is often made by our boss. The first point of contact for him, is usually the tool's website. And teampass' website hasn't been updated in a long time - at least it seems like it. e.g.

- system requirements are outdated
- link to docker hub -> The image hasn't been updated in five years
- footer copyright year is still 2021
- screenshots are from v2

Most people in management positions won't bother to look at GitHub to check for active development activities. And I guess this is not an unusual standpoint. 

I do not want to criticize, I really like Teampass - good job on that - just a head's up.

Keep up the good work!

**Comment by nilsteampassnet** [comment]
Hi @bjinthahouse 
Thank you for this message and interest🙏
You are absolutely right 👍
I need to spend time on it and update what needs to be updated 😉
Will do that 😅

**Reply by nilsteampassnet** [reply]
@bjinthahouse 
Site web is (mostly) updated ;)
